src/main/webapp/start.js

```diff
@@ -4,9 +4,10 @@ let startButtonCount = 0;
 // Takes the user to the lounge page when the start button is pressed
 async function startRoom() {
   if (startButtonCount < 1) {
+    await deleteRooms();
```

Comment:
Is the user signed in at this point? I thought they had to be signed in to delete some of these documents.

Reply:
I can move these methods to another point in our information flow, I just figured it would make the most sense to clear out all unused rooms when someone creates a new one